components/Steps/index.jsx

```diff
@@ -402,7 +402,7 @@ export const getInputProps = (
   { register, control } = {},
   errors
 ) => {
-  const { validation, ...props } = inputLabels[form][name];
+  const { validation, ...props } = inputLabels[form][name] || {};
```

Comment:
I remember doing this to fudge going straight to the submit screen yesterday. What is it for here? Make sure there is at least an empty object when the form hasn't been submitted yet?

Reply:
It's a fallback in case a label doesn't exist.
So, the label for `eligibilityCriteria.rateableLimitAnswerId` has been removed but it was returned by the BE (this has been fixed now). The problem we were having yesterday was related to this, `fixtures/dbMapping` had `"rateableLimitAnswerId": 1` (even if it wasn't passed anymore in the real UI), also this has been fixed.

In that file, there is an object called `inputLabels` that contains the properties of the form's input (name, type, validations, etc.). So, using our example, `inputLabels.eligibilityCriteria.rateableLimitAnswerId` was returning `undefined`, and you can't restructure undefined, so adding `{}` as a fallback solves the problem.